Add a link to the closure customization page (#5688)
filamentphp/filament: packages/forms/docs/03-fields.md

    ->getOptionLabelFromRecordUsing(fn (Model $record) => "{$record->first_name} {$record->last_name}")
```

> For more information on how colsure agruments work inside of Filament read [using closure customization](advanced#using-closure-customization).

Reviewer: ```suggestion
For more information on how closure arguments work in Filament, read [using closure customization](advanced#using-closure-customization).
```
